Recrutify/Recrutify.Services/Validators/CreateCandidateValidator.cs

```diff
@@ -27,16 +40,17 @@ public CreateCandidateValidator()
                 .NotEmpty()
                 .EmailAddress();
             RuleFor(c => c.Contacts)
-                .NotNull()
-                .NotEmpty();
-            RuleFor(c => c.Contacts)
-                .Must(c => c.Any(contact => contact.Type == Skype))
-                .WithMessage("Skype is required")
-                .Must(c => c.Count() <= 5)
-                .WithMessage("Maximum contacts reached");
+               .NotNull()
+               .NotEmpty()
+               .Must(c => c.Any(contact => contact.Type == Skype))
+               .WithMessage("Skype is required")
+               .Must(c => c.Count() <= 5)
+               .WithMessage("Maximum contacts reached");
             RuleForEach(c => c.Contacts)
                 .NotNull()
-                .NotEmpty();
+                .NotEmpty()
+                .ChildRules(x => x.RuleFor(x => x.Type).MaximumLength(50))
+                .ChildRules(x => x.RuleFor(x => x.Value).MaximumLength(50));
```

Comment:
Remove everything till .MaximumLength

Reply:
You do everything too complicated. Why you think child rule is necessary here? Why you don't write just MaximumLength, same as you wrote not null not empty